larq_compute_engine/core/BUILD

```diff
@@ -47,6 +47,7 @@ cc_library(
     hdrs = ["packbits_utils.h"],
     deps = [
         ":packbits",
+        "@org_tensorflow//tensorflow/lite/experimental/ruy/profiler:instrumentation",
```

Comment:
Just a reminder to ourselves: in lce-micro we have removed all this profiling stuff from the core utils (since its not in tflite-micro) and moved it elsewhere. So when we merge that, we shouldn't forget to take it out of these BUILD files again.